index.html

```diff
@@ -42,6 +42,7 @@
       <div class="inputBox">
         <label for="status">Status:</label>
         <select id="status">
+          <option value="none">None</option>
```

Comment:
There is going to be no status like `none`, we will not need it.
Instead we can have `blocked` for any task which is blocked because of some other task

Reply:
I will changed none to blocked.

Reply:
I will add screenshots of the before and after to the readme.